### ISSUE TITLE
Landmark accessibility links are cut off when new navigation is enabled

### ISSUE THREAD

**Issue by joelclimbsthings** [woocommerce/woocommerce-admin#6202]
**Describe the bug**
When the new navigation is enabled, the admin bar is hidden. This is resulting in the _landmark_ links provided for keyboard accessibility being cut off when displayed.

**To Reproduce**

- Enable new navigation
- Hit the tab key after loading the page until the landmark links are displayed
- Notice that they are not displayed correctly

**Screenshots**

![image](https://user-images.githubusercontent.com/444632/106060166-8a89f000-60a8-11eb-85fc-a765b9621323.png)

**Expected behavior**
The UI for the landmark links should be displayed correctly, and not cut off.
